### PR TITLE
literature: add seminars to lit detail page

### PR DESCRIPTION
**Pull request by monaawi**
Ref: #1053

### CODE REVIEW

**Comment by monaawi** [comment]
image

File: ui/src/literature/containers/DetailPageContainer/DetailPageContainer.jsx [hunk 1]
```diff
@@ -232,6 +242,21 @@ function DetailPage({ authors, record, referencesCount, supervisors }) {
...
+                    <TabNameWithCount
+                      loading={seminarsCount == null && loadingSeminars}
+                      name="Seminars"
+                      count={seminarsCount}
+                    />
+                  }
+                  key="3"
+                  forceRender
+                >
+                  <ContentBox className="remove-top-border-of-card">
```
Comment (harunurhan) [inline ui/src/literature/containers/DetailPageContainer/DetailPageContainer.jsx:256]: `className` is redundant

File: ui/src/literature/containers/LiteratureSeminars.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        metadata={result.get('metadata')}
+        selectedTimezone={LOCAL_TIMEZONE}
+        enableActions={false}
+      />
+    ),
+    []
+  );
+
+  return (
+    <Col span={24}>
```
Comment (harunurhan) [inline ui/src/literature/containers/LiteratureSeminars.jsx:42]: Is this really needed? it's strange to root of the component to be `Col` without `Row`

File: ui/src/seminars/components/SeminarItem.jsx [hunk 3]
```diff
@@ -32,27 +32,29 @@ function SeminarItem({ metadata, selectedTimezone }) {
...
-              icon={<FileOutlined />}
-              text="material"
-            />
-          )}
-          <ExportToCalendarAction seminar={metadata} />
-          {canEdit && (
-            <EditRecordAction pidType="seminars" pidValue={recordId} />
-          )}
-        </>
+        enableActions && (
```
Comment (harunurhan) [inline ui/src/seminars/components/SeminarItem.jsx:35]: problem with this approach (just hiding with flag), we deliver more data, and do more serializing than we need.. but I guess this is not significant at this point.

**Pull request merged**